### PR TITLE
Update Dockerfiles and add CI support

### PR DESCRIPTION
**Pull request by ZachPope531**
This removes CentOS 6 support, updates the EDM version for the remaining images, and adds `enthought.ci` tools to help automatic publishing

Right now I'm not sure how we log in to quay.io or if we even have a repository for these images to live in, so the `deploy` block in `.travis.yml` is commented out

### CODE REVIEW

**Comment by ZachPope531** [comment]
TODO: Get this repo a hatcher token

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Xenial is required for Docker
+dist: xenial
+
+language: python
+
+# Don't build anything but master, and PRs into master.
+branches:
+  only:
+    - master
+    - stable
```
Comment (ddebeau) [inline .travis.yml:10]: Let's remove `-stable` since we don't have a stable branch.

**Comment by ZachPope531** [comment]
Should be ready to push, @ddebeau final look-over?

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+echo $EDM_CONTAINERS_REGISTRY_SECRET | docker login -u $EDM_CONTAINERS_REGISTRY_ACCOUNT_ID --password-stdin quay.io
```
Comment (ddebeau) [inline deploy.sh:2]: NIT: `docker login -u $EDM_CONTAINERS_REGISTRY_ACCOUNT_ID -p $EDM_CONTAINERS_REGISTRY_ACCOUNT_ID quay.io` is a bit cleaner.

File: deploy.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+python -m ci publish-docker --yes
+docker logout quay.io
```
Comment (ddebeau) [inline deploy.sh:4]: No need to logout.

**Pull request merged**